zkevm-circuits/src/state_circuit/lookups.rs

```diff
@@ -104,6 +104,7 @@ impl<F: Field> Chip<F> {
             layouter.assign_region(
                 || format!("assign u{} fixed column", exponent),
                 |mut region| {
+                    region.name_column(|| format!("STATE_LOOKUP_fixed_u{}", exponent), column);
```

Comment:
The `u8, u10, u16, call_context_field_tag` are used as lookup columns.  It would be good to add lookup column annotation to those with `meta.cs.annotate_lookup_any_column()`.  I think you can add that in the `configure` function in line 89.

Reply:
fixed in [1b5aa41](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/1b5aa4193380414a8378031399be7a6eddfac11a)